external-crates/move/documentation/book/src/abilities.md

```diff
@@ -126,6 +130,16 @@ reachability rules for the abilities given above. If a struct is declared with t
 - `key`, all fields must have `store`.
   - `key` is the only ability currently that doesn’t require itself.
 
+An enum can have any of these abilities with the exception of `key` which enums cannot have since
```

Comment:
```suggestion
An enum can have any of these abilities with the exception of `key`, which enums cannot have because
```
If I didn't add the comma I wouldn't have made the "because" suggestion, but I was here. "Since" has more than one meaning, so I always suggest using "because" instead with ESL in mind.